great_tables/_formats.py

```diff
@@ -64,6 +64,7 @@ def fmt(
     fns: Union[FormatFn, FormatFns],
     columns: SelectExpr = None,
     rows: Union[int, List[int], None] = None,
+    is_substitution=False,
```

Comment:
I added this flag mostly because it's very fast, but am down for creating a dedicated `sub()` function, if it seems better!

Reply:
What you have makes a lot of sense! By way of comparison, the R version has the analogous `subst()` function but it's not exported.